packages/components/addon/components/hds/interactive/index.ts

```diff
@@ -6,7 +6,25 @@
 import Component from '@glimmer/component';
 import { action } from '@ember/object';
 
-export default class HdsInteractiveComponent extends Component {
+interface HdsInteractiveSignature {
+  Args: {
+    href?: string;
+    isHrefExternal?: boolean;
+    isRouteExternal?: boolean;
```

Comment:
nitpicking: can we keep the arguments in the same order as in the `Button` so it's easy to compare?

Reply:
not needed anymore as we now have consistency by extending `HdsInteractiveSignature` – I initially followed the order of the component API in our docs, hence the mismatch